ray-operator/config/rbac/leader_election_role.yaml

```diff
@@ -30,3 +30,12 @@ rules:
   - events
   verbs:
   - create
+- apiGroups:
```

Comment:
This is necessary for Leader Election. We can refer to [leader_election_role.yaml](https://github.com/kubernetes-sigs/kubebuilder/blob/master/docs/book/src/cronjob-tutorial/testdata/project/config/rbac/leader_election_role.yaml) in the kubebuilder repo for further.